README.md

```diff
@@ -22,6 +28,8 @@ Activate the virtual environment:
 
     $ . env/bin/activate
 
+On OSX do `. venv/bin/activate` instead.
```

Comment:
D'oh! That's not *just* on OSX. That's the command everywhere, and the instructions ahve just been wrong until now. :)